recipe/meta.yaml

```
@@ -12,51 +15,82 @@ source:
   sha256: {{ checksum }}
 
 build:
-  number: 2
+  skip: true            # [win32]
+  skip: true            # [win and py<35]
```

They are not needed, we don't build for these platform anymore.
```suggestion
  skip: true            # [win32]
  skip: true            # [win and py<35]
```

---

Fixed.